src/renderer/scss-partials/_ft-list-item.scss

```diff
@@ -148,11 +148,20 @@ $watched-transition-duration: 0.5s;
       margin-inline-start: 4px;
     }
 
-    .favoritesIcon {
-      font-size: 17px;
+    .playlistIcons {
       justify-self: end;
       margin-inline-end: 3px;
       margin-block-start: 3px;
+
+      display: grid;
+      grid-auto-flow: column;
+      justify-content: flex-end;
+    }
+    .addToPlaylistIcon,
+    .trashIcon,
+    .upArrowIcon,
+    .downArrowIcon {
+      font-size: 17px;
       block-size: fit-content;
```

Comment:
```suggestion
      justify-content: flex-end;
      block-size: fit-content;
    }
    .addToPlaylistIcon,
    .trashIcon,
    .upArrowIcon,
    .downArrowIcon {
      font-size: 17px;
```
This bug was re-introduced. Here's the fix.

![Screenshot_20231118_220012](https://github.com/FreeTubeApp/FreeTube/assets/84899178/2f5bd36c-1016-4b4e-b5fa-a7e0f8abea4a)

Reply:
(Test in your own branch first - I tend to make minor linting mistakes on the Web UI - and you can get rid of the `block-size: fit-content;` on line 165 as well)

Reply:
fixed